code/game/objects/structures/beds_chairs/alien_nest.dm

```diff
@@ -48,7 +48,7 @@
 			unbuckle_mob(M)
 			add_fingerprint(user)
 
-/obj/structure/bed/nest/user_buckle_mob(mob/living/M, mob/living/user)
+/obj/structure/bed/nest/user_buckle_mob(mob/living/M, mob/living/user, check_loc = FALSE)
```

Comment:
Why does this one not check loc when all the others do? Is there any reason for it?

Reply:
because it's basically a bed, it's to make it convenient for xenos to buckle victims adjacently

Reply:
nevermind, i gotta change it so that it does